docs/components/checkboxes/README.md

```diff
@@ -3,7 +3,7 @@
     "title": "Checkboxes",
     "layout": "LayoutComponent",
     "title_metadata": "CdrCheckbox, cdr-checkbox",
-    "summary": "Permits user to make one or more selections from a list",
+    "summary": "Permits user to make one or more selections from a list.",
```

Comment:
I thought we had edits removing these terminals?

Reply:
Hm, I changed that based on the edit in the google doc and to match a couple others that I spot checked (though I see now not all of them have it). Will ask Pat

Reply:
Ok, Pat confirmed - no terminal punctuation on these intros.